packages/server/src/handlers/BaseHandler.ts

```diff
@@ -149,6 +149,16 @@ export class BaseHandler extends EventEmitter {
         reject(err.name === 'AbortError' ? ERRORS.ABORTED : err)
       })
 
+      const postReceive = throttle((offset: number) => {
+        this.emit(EVENTS.POST_RECEIVE_V2, req, {...upload, offset})
+      }, this.options.postReceiveInterval)
```

Comment:
Should we specify the leading or trailing options here to get the behavior that is mentioned in the docs? https://lodash.com/docs/#throttle Or do the defaults already provide the behavior we need?

Reply:
I think I prefer the defaults, which means the first event is immediately fired and if another event comes in within the wait period, it's fired after the timeout.

Reply:
Ok, but is the documentation correct then? It currently says 

> This means you are not guaranteed to get (all) events for an upload. For instance if
`postReceiveInterval` is set to 1000ms and an PATCH request takes 500ms, no event is emitted.

But if we also trigger the event on a leading edge, we would get an event at 0m, wouldn't we?

Reply:
Good catch. Perhaps the leading edge is not as valuable, which would be fired almost instantly (and we have `onIncomingRequest`) 🤔 If we set `leading` to `false` the docs are correct.

Reply:
> If we set `leading` to `false` the docs are correct.

Sounds good 👍